# My Jupyter Notebook on IBM Watson Studio

**Muhamad Ihsan Fauzi**

Product Manager Data Platform

_I am interested in Data Science because I want to understand how to uncover insights from data_

### In this project I'm trying to load sample car data-sets in csv format to pandas data frame and display the top 5 rows

In [1]:
import pandas as pd
df = pd.read_csv("data.csv")
df.head()

Make       Model  Year             Engine Fuel Type  Engine HP  \
0  BMW  1 Series M  2011  premium unleaded (required)      335.0   
1  BMW    1 Series  2011  premium unleaded (required)      300.0   
2  BMW    1 Series  2011  premium unleaded (required)      300.0   
3  BMW    1 Series  2011  premium unleaded (required)      230.0   
4  BMW    1 Series  2011  premium unleaded (required)      230.0   

   Engine Cylinders Transmission Type     Driven_Wheels  Number of Doors  \
0               6.0            MANUAL  rear wheel drive              2.0   
1               6.0            MANUAL  rear wheel drive              2.0   
2               6.0            MANUAL  rear wheel drive              2.0   
3               6.0            MANUAL  rear wheel drive              2.0   
4               6.0            MANUAL  rear wheel drive              2.0   

                         Market Category Vehicle Size Vehicle Style  \
0  Factory Tuner,Luxury,High-Performance      Compact         Coupe   
1                     Luxury,Performance      Compact   Convertible   
2                Luxury,High-Performance      Compact         Coupe   
3                     Luxury,Performance      Compact         Coupe   
4                                 Luxury      Compact   Convertible   

   highway MPG  city mpg  Popularity   MSRP  
0           26        19        3916  46135  
1           28        19        3916  40650  
2           28        20        3916  36350  
3           28        18        3916  29450  
4           28        18        3916  34500

---
~~This sentence is intentionally put here to separate between output and bottom markdown~~

>```python
>import pandas as pd
>df = pd.read_csv("data.csv")
>df.head()
>```

The actions performed in the code above are:
1. Importing the required library (in this case pandas library)
2. Loading data into pandas data frame
3. Showing the top 5 rows

Some important features can be viewed from this data set are
* Miles Per Gallon (MPG)
* Engine HP
* Price

Sample table containing important features only from BMW Car Maker:
|  MPG  |Horse Power| Car Price |
|-------|:---------:|----------:|
|26     |335        |46135      |
|28     |300        |40650      |
|28     |230        |34500      |

![alt text](https://seeklogo.com/images/B/bmw-logo-248C3D90E6-seeklogo.com.png "BMW Logo")

the data set used in the code above can be downloaded from [here](https://www.kaggle.com/CooperUnion/cardataset)